content/en/tracing/setup_overview/compatibility_requirements/dotnet-core.md

```diff
@@ -20,7 +20,11 @@ further_reading:
 
 - The .NET Tracer supports all .NET-based languages (for example, C#, F#, Visual Basic).
 
-- The .NET Tracer supports automatic instrumentation on .NET 5, .NET Core 3.1, and .NET Core 2.1. It also supports [.NET Framework][1].
+- The .NET Tracer supports automatic instrumentation on:
```

Comment:
This "Compatibility" section and the one in `dotnet-framework.md` should be mostly the same, except that this one should list specific version of .NET Core/5. Some differences that we should fix (maybe a separate PR):

- `dotnet-core.md` mentions other .NET languages (C#, F#, Visual Basic), but `dotnet-framework.md` does not
- `dotnet-framework.md` says the tracer supports both automatic and custom instrumentation. `dotnet-core.md` only mentions custom instrumentation in the context of keeping nuget and msi package version in sync
- [nit-picking] `dotnet-framework.md` links the the github repository first, `dotnet-core.md` lists it last

Reply:
Thanks for pointing that out - @kayayarai if we could address these that would be great!